buy-shirt.cy.ts

```diff
@@ -0,0 +1,24 @@
+describe("Buy a t-shirt", () => {
```

Comment:
Please create this test in cypress/e2e folder

Reply:
done @veronicatofino